### PR TITLE
SOR-1955 update github.com/golang/protobuf/protoc-gen-go to latest

### PR DESCRIPTION
**Pull request by vtopc**
https://jira.namely.land/browse/SOR-1955

### CODE REVIEW

**Review by alex-namely — APPROVED** [review]

**Review by ido-namely — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -83,10 +83,7 @@ RUN go get -u github.com/coinbase/protoc-gen-rbi
 RUN go get github.com/gomatic/renderizer/cmd/renderizer
 
 # Origin protoc-gen-go should be installed last, for not been overwritten by any other binaries(see #210)
-RUN set -e && \
-  GO111MODULE=on go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.5 && \
-  cd /go/pkg/mod/github.com/golang/protobuf@v1.3.5/protoc-gen-go && \
-  go install .
+RUN go get -u github.com/golang/protobuf/protoc-gen-go
```
Comment (abe545) [inline Dockerfile:86]: Why do we no longer need to run `go install .`?

Comment (vtopc) [reply Dockerfile:86]: `go get -u` in GOPATH mode is doing `go install` under the hood, but it's not supporting `path@version` so had to do that workaround in the #211

**Review by abe545 — APPROVED** [review]

**Comment by ido-namely** [comment]
Hey @vtopc ,
can we merge this?

**Comment by vtopc** [comment]
> Hey @vtopc ,
> can we merge this?

I guess, but it will break go-common(at least unit test)

**Pull request merged**